### PR TITLE
Bug 968298: Remove cruft from devhub and move all docs to MDN.

### PR DESCRIPTION
**Pull request by kmaglione**

### CODE REVIEW

File: apps/amo/cron.py [hunk 1]
```diff
@@ -84,17 +84,6 @@ def gc(test_result=True):
...
-        log.debug('Cleaning up old packaged add-ons.')
-
-        cmd = ('find', settings.PACKAGER_PATH,
-               '-name', '*.zip', '-mtime', '+1', '-type', 'f',
-               '-exec', 'rm', '{}', ';')
-        output = Popen(cmd, stdout=PIPE).communicate()[0]
-
-        for line in output.split("\n"):
-            log.debug(line)
-
```
Comment (magopian) [inline apps/amo/cron.py]: Is that obsolete? I don't find anything about this removal in the bug

Comment (magopian) [inline apps/amo/cron.py]: Oh, `Remove the 'Add-on Packager' link`, I see ;)

File: apps/stats/templates/stats/csv_header.txt [hunk 2]
```diff
@@ -1,12 +1,8 @@
 {% if title %}
 #    {{ title }}
 {% else %}
-{% if settings.MARKETPLACE %}
-#    marketplace.firefox.com Statistics for app {{ addon.name }}
-{% else %}
```
Comment (magopian) [inline apps/stats/templates/stats/csv_header.txt]: \o/

File: apps/devhub/views.py [hunk 3]
```diff
@@ -1640,32 +1582,51 @@ def admin(request, addon):
...
+            'policies/contact': 'AMO/Policy/Contact',
+        })
+    if waffle.switch_is_active('mdn-agreement-docs'):
+        # This will most likely depend on MDN being able to protect
+        # pages.
+        mdn_docs.update({
+            'policies/agreement': 'AMO/Policy/Agreement',
+        })
+
+    all_docs = {'policies',
```
Comment (magopian) [inline apps/devhub/views.py]: As you might have guessed from the travis results, this is a syntax error in py26 (the `set literals` is only backported to 2.7 from what I can tell).

As it's not very important to have a `set` here, i'd swith to a simple list or tuple.

**Comment by magopian** [comment]
This is huge! Not sure who could give you a good review of this, @clouserw any idea?

Could you please fix the `set literals` issue so we see what travis has to say?

Thanks for this impressive work!

File: apps/devhub/forms.py [hunk 4]
```diff
@@ -757,206 +757,6 @@ def render(self):
         return mark_safe(''.join(force_unicode(w) for w in self))
 
 
-class PackagerBasicForm(forms.Form):
```
Comment (clouserw) [inline apps/devhub/forms.py]: This closes bug 922101

File: apps/devhub/templates/devhub/builder.html [hunk 5]
```diff
@@ -1,66 +0,0 @@
...
-      complete, we make it easy to share it with over 400 million Firefox users
-      world wide.
-    {% endtrans %}
-    </p>
-  </div>
-</div>
-
-<div class='island'>
-  <div class='jetpack-sdk'>
-    <img src="{{ static('img/developers/jetpack_icon_command.png') }}" alt="">
```
Comment (clouserw) [inline apps/devhub/templates/devhub/builder.html]: should remove the file from /static/.  Could probably remove jetpack* in there.

File: apps/devhub/templates/devhub/docs/case-studies-cooliris.html [hunk 6]
```diff
@@ -1,110 +0,0 @@
...
-
-<h2>Case Study: Immersive 3-D Experience with Cooliris</h2>
-<div id="casestudy" class="primary full" role="main">
-{{ sidebar() }}
-
-{# No trans tags because we aren't localizing this yet, per remora comments. #}
-
-<div class="info">
-  <ul class="blocks">
-    <li class="item"><img src="/media/img/docs/case-studies/cooliris-wordmark.png" alt="Immersive 3-D Experience with Cooliris"/></li>
```
Comment (clouserw) [inline apps/devhub/templates/devhub/docs/case-studies-cooliris.html]: same with these images.  Looks like a bunch of static assets could go with this patch

File: apps/devhub/templates/devhub/nav.html [hunk 7]
```diff
@@ -59,12 +51,14 @@ <h1 class="site-title prominent">
       </ul>
     </li>
     <li>
-      <a href="#" class="controller">{{ _('Community') }}</a>
+      <a href="#" class="controller">{{ _('Community & Support') }}</a>
```
Comment (clouserw) [inline apps/devhub/templates/devhub/nav.html]: not sure if that comes through as &amp; or not on the page

**Comment by clouserw** [comment]
I read through it and added some comments.  overall I like it a lot

File: apps/devhub/templates/devhub/nav.html (hunk 7 above)
Comment (kmaglione) [inline apps/devhub/templates/devhub/nav.html]: Huh. I was expecting it to be escaped, but it's not.

**Comment by kmaglione** [comment]
Removed stale assets and set literals. Added second bug# to summary.

**Comment by kmaglione** [comment]
Fixed failed tests.

**Comment by clouserw** [comment]
I'm happy if @magopian is happy. r+ from me. Thanks for doing this.

**Pull request merged**